packages/ui/src/components/va-viewer/VaViewer.demo.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <VbDemo class="flex flex-wrap space-x-4 space-y-4">
+    <VbCard title="Default">
+      <va-viewer>
+        <va-image
+          style="--va-image-position: relative;"
+          :src="getImagePath(600, 800)"
+          :max-width="200"
+          fit="contain"
+        >
+          <template #loader>
+            Loading...
+          </template>
+        </va-image>
+      </va-viewer>
+    </VbCard>
+
+    <VbCard title="Custom anchor">
+      <va-viewer>
+        <template #anchor="{ openViewer, isOpened }">
```

Comment:
Let's say that. If there is no default slot, we use anchor and image slots. If there is default slot, we  use default slot as anchor and image.

So there is a simple interface:
```vue
<va-viewer><va-image/></va-viewer>
```

or a bit complicated:
```vue
va viewer
  va button #anchor="{ show }" @click="show"
  va image #image
```